### PR TITLE
Create button component

### PR DESCRIPTION
**Pull request by nikitha2000**
create button component

### CODE REVIEW

File: src/component/Button.js [hunk 1]
```diff
@@ -1,3 +1,21 @@
 
- 
+import React from "react"
+
+const Button = ({ color, label, onClick, isClicked, isDisabled}) => {
+   
+
+
```
Comment (vonnue-nidheesh) [inline src/component/Button.js:7]: Remove empty lines.

File: src/component/Button.js [hunk 2]
```diff
@@ -1,3 +1,21 @@
 
- 
+import React from "react"
+
+const Button = ({ color, label, onClick, isClicked, isDisabled}) => {
```
Comment (vonnue-nidheesh) [inline src/component/Button.js:4]: Color, isClicked, and isDisabled should be optional props.

Comment (vonnue-nidheesh) [reply src/component/Button.js:4]: `const Button = ({ color='SomeColorCode', label, onClick, isClicked=false, isDisabled=false}) => {`

This way the props are optional. If we do not pass the value for these props it will automatically set these default values as props value.

File: src/component/Button.js [hunk 3]
```diff
@@ -1,3 +1,21 @@
```
Comment (vonnue-nidheesh) [inline src/component/Button.js:1]: The file extension should be `jsx`.

File: src/App.js [hunk 4]
```diff
@@ -7,7 +7,24 @@ import Button from './component/Button';
 
 
 function App() {
-
+  const [clickedButton, setClickedButton] = useState(null)
```
Comment (vonnue-nidheesh) [inline src/App.js]: The default value should be `false` right?

File: src/App.js [hunk 5]
```diff
@@ -7,7 +7,24 @@ import Button from './component/Button';
```
Comment (vonnue-nidheesh) [inline src/App.js:8]: PR title should be something like 'Create button component'.

File: src/component/Button.js (hunk 2 above)
Comment (nikitha2000) [reply src/component/Button.js:4]: ok

File: src/App.js [hunk 6]
```diff
@@ -1,17 +1,40 @@
...
+import ToDoForm from "./component/TodoForm";
 
 
 
 
 
 function App() {
-
+  const [clickedButton, setClickedButton] = useState(false)
+
```
Comment (vonnue-nidheesh) [inline src/App.js:12]: Remove extra line

File: src/App.js [hunk 7]
```diff
@@ -1,17 +1,40 @@
...
-
+  const [clickedButton, setClickedButton] = useState(false)
+ 
+
+  const handleClick =  (buttonId) => {
+     setClickedButton(buttonId);
+  }
+
+ 
+
```
Comment (vonnue-nidheesh) [inline src/App.js:19]: Remove extra lines

File: src/App.js [hunk 8]
```diff
@@ -1,17 +1,40 @@
...
+        </div>
+      </div>
+    );
 }
 
 export default App;
 
-                 
+
```
Comment (vonnue-nidheesh) [inline src/App.js:38]: Revert this change

File: src/component/Button.js [hunk 9]
```diff
@@ -1,3 +1,21 @@
 
- 
+import React from "react"
```
Comment (vonnue-nidheesh) [inline src/component/Button.js:2]: Remove this file.

File: src/component/Button.jsx [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+            isDisabled={isDisabled}
+        >
+            {label}
+        </button>
+    )
+}
+
+export default Button
+   
+
```
Comment (vonnue-nidheesh) [inline src/component/Button.jsx:17]: Remove extra line

File: src/App.js [hunk 11]
```diff
@@ -1,17 +1,40 @@
...
+   
+    return (
+      <div className="App">
+        <div className='app-container'>
+          <h1>TODO App</h1>
+            <div className='button-container'>
+               <Button color="#8888b9" label="Todo" onClick={() => {handleClick("button1")}}  isClicked={clickedButton === "button1"}  />
+               <Button color="#8888b9" label="Todo list" onClick={() => {handleClick("button2")}}  isClicked={clickedButton === "button2"} />
+            </div>
+
```
Comment (vonnue-nidheesh) [inline src/App.js:29]: Remove extra line

File: src/App.js [hunk 12]
```diff
@@ -1,17 +1,40 @@
 import React, { useState } from "react"
 import './App.css';
 import Button from './component/Button';
+import ToDoForm from "./component/TodoForm";
```
Comment (vonnue-nidheesh) [inline src/App.js:4]: Sort imports. import nearest files last

eg:
```
import React, { useState } from "react"
import Button from './component/Button';
import ToDoForm from "./component/TodoForm";
import './App.css';
```

Comment (vonnue-nidheesh) [inline src/App.js:4]: The toDOForm component is not used anywhere. Please remove this import.
